src/entrypoints/FieldAddon/FieldAddon.tsx

```diff
@@ -117,6 +120,16 @@ export default function FieldAddon({ ctx }: Props) {
               )
               break
             }
+            case TranslationFormat.seo: {
+              const field: any = get(ctx.formValues, `${fieldPath}.${locale}`);
+              translatedField = {
+                title: await getTranslation(translatableField.title, options),
+                description: await getTranslation(translatableField.description, options),
+                image: field?.image,
+                twitter_card: field?.twitter_card,
```

Comment:
Not sure what is the best approach for this fields, they are only on the `formValues` if they have already been saved, but not when modified and not saved. It could also default to the ones from the default locale.

Reply:
We can just see if images/twitter_cards are already added and if not get the one from the defaultLocale

So do this:
```javascript
case TranslationFormat.seo: {
   const currentField: any = get(ctx.formValues, `${fieldPath}.${locale}`)
   const defaultField: any = get(ctx.formValues, `${fieldPath}.${locales[0]}`)

   translatedField = {
         title: await getTranslation(translatableField.title, options),
         description: await getTranslation(translatableField.description, options),
         image: currentField?.image || defaultField?.image,
         twitter_card: currentField?.twitter_card || defaultField?.twitter_card,
   }
   
   break
}

Reply:
Great, I have added that. I think `translatableField` is the same as the `defaultField` you defined, isn't it? Otherwise let me know and I will change it to your suggestion.

Reply:
Yes, awesome. That should indeed be the same!